lib/ash/actions/destroy/bulk.ex

```diff
@@ -297,6 +297,10 @@ defmodule Ash.Actions.Destroy.Bulk do
 
               %{bulk_result | notifications: notifications}
             else
+              Ash.Actions.Helpers.warn_missed!(atomic_changeset.resource, action, %{
```

Comment:
@zachdaniel I'm not sure about this. do we actually want to just throw away notifications here?

Reply:
Yeah, we warn that we couldn't send them and then toss them, seems right to me 👍